src/test/features/response/form/models/responseType.ts

```diff
@@ -0,0 +1,28 @@
+import { expect } from 'chai'
+import { ResponseType } from 'response/form/models/responseType'
+
+describe('ResponseType', () => {
+
+  describe('all', () => {
+
+    it('should return array of 3', () => {
```

Comment:
this test doesn't add value imo, it means you have to change it if you add a response type but why does the test care?

Reply:
Yeah, that does not protect use from removing one of existing states and replacing it with new one.